Model/src/GamePackage/Bank.java

```diff
@@ -1,7 +1,158 @@
 package GamePackage;
 
-/**
- * Created by Dylan Kernohan on 2/10/18.
- */
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+
+import java.util.ArrayList;
+
+/****************************************************************************************************************
+ * The bank class is responsible of keeping track of the game's houses, player montages, and also perform
+ * certain transactions.
+ *
+ * @author Santiago Quiroga
+ * @version 2/13/2018
+ ***************************************************************************************************************/
 public class Bank {
+    /* Number of houses and Hotels */
+    private int numHouses, numHotels;
+
+    /* List of properties owned and properties mortgaged */
+
+    private ArrayList<PropertySquare> propertiesOwned, propertiesMortgaged;
+
+    /**
+     *  This constructor initializes the bank with a specific set of properties,
+     *  and uses the size of properties tocompute the size of the properties that can be mortgaged.
+     * @param listOfProperties
+     */
+    public Bank(ArrayList<PropertySquare> listOfProperties) {
+        /** initialises all variables */
+        propertiesOwned = listOfProperties;
+        propertiesMortgaged = new ArrayList<>(listOfProperties.size());
+        numHotels = 16;
+        numHouses = 16;
+    }
+
+    /****************************************************************************************************************
+     * This method will return a specific property, based of its ID.
+     *
+     * @param property
+     ***************************************************************************************************************/
+    public PropertySquare giveProperty(PropertySquare property) {
+        //removes and returns the specific property.
+        return propertiesOwned.remove(property.getId());
+    }
+

```

Comment:
Will we want to make this deal with type ownableSquares when we make that? Instead of PropertySquare?

Reply:
add this to an issue for the OwnableSquare